plugins/automation/admin/assets/javascripts/discourse/controllers/admin-plugins/show/automation/edit.js

```diff
@@ -27,6 +27,26 @@ export default class AutomationEdit extends Controller {
     }
   }
 
+  get disableEnabledToggle() {
+    return !(this.automation.canBeEnabled || this.automation.enabled);
+  }
+
+  @action
+  async toggleEnabled() {
+    const automation = this.model.automation;
+    // automation.set("enabled", !automation.enabled);
+    automation.enabled = !automation.enabled;
+    this.set("isUpdatingAutomation", true);
```

Comment:
This property is used to control the loading state of the automation form's submit button. Therefore, toggling the enable automation button is causing the form's submit button to briefly jump as it displays a "loading" icon before disappearing.

Since this button is a toggle, I personally think we can just remove any sort of loading indicator.